examples/tutorials/03_reusable_components.ipynb

```diff
@@ -0,0 +1,1845 @@
vrjXzzG99iZGSEWBkPKk+CRqG1spOyPMcHh9iw4Tl27dxtxQKD1lAoFBgfG2fg8ABPPvkkR44cYWRkJI2QANJ2SOvHYsrW2ypTIkkdSaJKzHaXXnopmzdtZsaMGda/SU/oX2Htj3p7e3n22WfZtGkzpVIH8+cv4Etf+grPPfcCDz30MFu2vMzxwSHuu+9+Dh06TKlUYnx8nK6uLoqlEj/4wQ/YvHlzOlnN+sYnkR0rVqxg165deJ7PggUL0hS/JMJIaUUSYdTX18cLzz/Pc889x/DIMHEc2apvRubyPJ/h4WG2bt3GU089xeDgIHGkTDlzK8RoWy5cCEkYJiKdZ6uMNfpfJT5KwoopbW3tzJu3gPvvv5/Nmzdz2223MW/efGbO6E+jddIoKo2NmNHkcj6xFWs8T3LgwAE2b97M5s0vMTZWRimB5+XSSnNCeOTzeQYHB9m4cSP79u3PCG5JVT9l+9BM/qvVCgcO7OcHP/gBY2Nj1Go1Lr74YjZseI77HzBRTXfffRdxHDNjxgw2b97M888/z7Fjg6ZKoapXSzPXSqZjPLmXpPBIqmMJJO3tHZxzzjnce+99bNy4ka997WssWbKE2bNns2jRIu688042bNjA888/TxTFTJ8+ixUrVvG1r32d9euf4+Hv/4D1z64nF+Tp6OjiuedeYGDgSGYs1sdkIvYdOXKUbdu288ILGxkfLxOGkb0HjCiSz+dRSjFnzhw8z+OOO+7g2WefZcuWLcRxlIqRpjqcqV7oeR4zZ86gs7OTr3zlq7z44otsWL8ez/OYPn06uVyOhx56iPXr13PvvfemhtPz58+nUChwzTXX0tHexRWXX0VXVw+zZvabqEQt8Kyoq2Jto3dGuP/++zl8+DDFYpEwDGlvbyefz/PII4+wadOm9Lsgub87OjpZtHAR3/rWrTzzzHoeffRRnn/+eRtl53H48CE2PPc827btYGRk1IqXxvfNHLcuiCfjBiSzZvVTqVSNsCuMCDg2Nsb06TNSodgFEDkcjixOMHI4HA6Hw/EaYkUMrRkYOMyGDc/yd3/3N3zpS19kzpzZfOxjH2Pr1q18+9vfZnR0FCklQRDwnve8h3K5zHe/+12OHT2SET9sBIxWbNq0ic7OLt785rdy44030dc3neeff54wrKWTwaxR+CWXXMLChQu47bbb2LFjB1dcfiXvfe9P8fDDP+A7d9xJFMZ40mdwcJA77/wuhw4N8N73/hSFQoF3v+c9VMoV7rjjDo4cOWKickxWkJ1oazZsWE9HRztvectbue66tzB//gKefvppKxBhI55supISLFu6lDe96Xruufsenn32WQqFEjNnzkyjlhYuXMwVV1zJt771bSrlKje//R1s27YdgWDO7Lm0ldrwvYC5c+bhSR+BpLurh+nTZ9AgfKR9ZoSgrq4uFixYwJVXXGknk0zo32T9d73rXVSrNW6//Q5qtZCP/vLH6Ozs5utf/wbPP/c8+VwBISR79uzh29++lRkzZvHmN7+V7u5uPviBD/Lss89y7333UK1WgGSiaoQTbV3AZ82aRU93N1dffRWlUhGlYhPxJDRKq1TAKBTyvO1tbwMEd37nuxwZOMqsWbNpb+8kSYt833t/iiMDR7n99ju44IKLWLhwEQMDAxQKBWbPnk0ul6O3t5dp06YBkM/nmTNnjh2jjeMFzIS9WGyjv78f3zcRGR/+8IeR0uMLX/gCvh/w0Y/+MqVSGzNmzKSjoxMhTNUt6QkzToQgikK0junvn8X73vc+7r33Ph763kN84P0fJKxFVMpV+mfNplAo4Hsm4mPp0mVcd9313HHHnaxfvx4pPfr7Z5PP59FK0N/fT3tbOyqG81av4bzV5/PlL32FOFJcddXVHDp4mPNWr+HDP/MzPP3UU9xxxx2Mjo6iteZtb3sbbW1t3Hrrrezdu8dGywmExJamNxEuc+bMpVRqRynNtGnT6e3tRWud9mdbWxs//aGfQSv48pe+TKnYxs/97M9RKrbzsx/+OXq6e7n127dx8MBBZvfPxfdy/MLP/yLzz1nAbbfeztNPr6dQKKE1vP3tN9PW1sHOnbvsFWi8Fu3t7fz0h36G9es38J077uStb7mBXFAgjhRzZs81qXPSo1Rso3/WbPK5Ah//tV/n2NFjfPfOu4jCmLlz5mYi13yEgDiOUCqip6eXj370oxw/fpxvfetbxJFixvSZdHZ08csf/WUOHjjEbbfexpLFS7nlPe/Fkx59fdN597vew9rzL0ApuPTSy7nhbTfS3d2LlB6zZvWTz5cIAo/Ozi5mzpyJFJId23fwrW9+mzmz53LVlVfR093LB97/QZ5+6hkeuP9B4kjZtvoIISgUCvzKr/wK03p7+fa3v83TTz9Le1snhUKRW265hccee5xnn32Wm29+J0NDI6gY5s09h1yQR0pJX98MOjq6kFLQ3t7BrP45xFqzcNEiFIJaFKKBUMUcGjjC+esuAE8msYYOh8ORIrJ/3XqteeaZZ14EVhUKBVatWokgplzZx5ZXHmP50qUUi53UyzXaKhWuZKOjAfvjwVYy0dhKGiJmZHgfO3cfZuXy6wiCaTSWx51sX6Ps2vUUUTzOooVL7F/tkvKhZMZfEo/rNFXH64tOwv21h9KS//wnf44ni/zHP/k12toC0MZLQIjEWNiNyTcqWsPWV/bxp//5z/n5n/8Qb7/pzSBq1KtYSRrCNX7caI/aSC/3fPd5rrn+SiKpUdIIKVopnn3mGaqVMn19PTzywx9w8OAAv/2J36Orq8uYqwpJrGJbeUynPuBaKZ5Z/yyvvPKyPZA5Z9/3ufLKK21qlE6FiN27d/Pkk08SRZGN/jHeOcuWLWPdugvSiAmTJmOTsLSup29hE7K0MSkWUpry2xKUNqkqu3bt5oknnkjbIoSJ3LnoogtZsmSJiRRRRuw4duwIDzxwfxoRpLVJN1u8aCkXXnghUoK0XktxrEx6VBIhRGIQbUUWm+IkZBLZJG3bTRSOJz2kFAwNDf
```

Comment:
I feel like we're missing one more line here ... something to summarize it all and tie into the Lineapy theme of getting to production quickly like: 

As demonstrated in this tutorial, Lineapy re-usable components can streamline a lot of data science and engineering work allowing the modern data team to define and modify data workflows quickly and flexibly. This helps Data Scientists and Data Engineers move these workflows to production and see impact in a faster, more reliable, and safer way. 

Please edit the above so that it fits with the rest of the notebook.